x-pack/plugins/lists/server/routes/utils/build_siem_response.ts

```diff
@@ -49,7 +49,7 @@ export class SiemResponseFactory {
   constructor(private response: KibanaResponseFactory) {}
 
   // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
-  error<T>({ statusCode, body, headers }: CustomHttpResponseOptions<T>) {
+  error<T>({ statusCode, body, headers, bypassErrorFormat }: CustomHttpResponseOptions<T>) {
```

Comment:
bypassErrorFormat is an option already provided by `CustomHttpResponseOptions` I have just added an implementation